src/main/scala/monitors/TLMonitorInjToDcache.scala

```diff
@@ -58,6 +58,28 @@ case class AopTestModule (wide: Int) extends Module {
   io.random := 0.U
 }
 
+case object MonitorPrintPrefix extends Field[Option[String]](None)
+
+object BundlePrintTest {
+  def print_bundle(bundle: TLBundle, edge: TLEdge) {
+    edge.params(MonitorPrintPrefix).foreach { prefix =>
```

Comment:
is `MonitorPrintPrefix` actually set here? i think you should just take `prefix` as a parameter to `print_bundle`